src/components/SocialLinks-Color/index.js

```diff
@@ -51,6 +51,7 @@ const SocialLinksColor = () => {
               height="30px"
               src={github_icon}
               alt="github"
+              style={{ filter: 'grayscale(1.5) invert(0.38)' }}
```

Comment:
Remove the style from here, and add the CSS on [this](https://github.com/layer5io/layer5/blob/6e119c6eb52e3bfa12a74fd2bb57bdef437d5954/src/components/SocialLinks-Color/sociallinkscolor.style.js#L12) line like
```
.github {
    filter: 'grayscale(1.5) invert(0.38)';
}
```

Reply:
yes i will but previously you was telling that to change something in hover effact. i don't think it is needed to change in hover  effact if it is needed then can you please specify.

Reply:
@Nikhil-Ladha i have done the complete changes that are required and suggested by you. Can you now review this pr.